### DISCUSSION TITLE
Cannot able to change the default component colorscheme.

### DISCUSSION THREAD

**Discussion by nishu-murmu** [chakra-ui/chakra-ui#7345] [Q&A]
Is this the write method to configure the button component?
I'm using Next.js btw.
![colorscheme](https://user-images.githubusercontent.com/71123337/218139009-06137c22-2cbd-4f03-812b-38a490ee8dae.png)

**Comment by keithburgie** [comment]
That's not quite how you'd use `colorScheme`. If you're trying to change the text color to look good on your background, just use `color` instead:
```typescript
baseStyle: (props: StyleFunctionProps) =>({
    background: mode('black', 'white')(props),
    color: mode('teal.800', 'teal.200')(props),
  }),
```

`colorScheme` sets multiple values, like `bgColor`, `color`, `_hover` colors, etc, when you use it like `<Button colorScheme="teal" />`. It uses the color token and not the shade (800, 200, etc.)

You could try one of these two options and see if it gets you closer to your goal:
```typescript
baseStyle: (props: StyleFunctionProps) => {
    const {colorScheme: c} = props;
    return {
      background: mode('black', 'white')(props),
      color: mode(`${c}.800`, `${c}.200`)(props),
    };
  },
  
  baseStyle: (props: StyleFunctionProps) =>
    props.theme.components.Button.variants.solid({
      colorScheme: 'teal',
    }),
```
